rebar.config

```diff
@@ -0,0 +1,28 @@
+{erl_opts, [debug_info]}.
+{deps, [
+    telemetry
```

Comment:
What does this dependency mean? Is this `~> 0.4`? `>= 0.4`?

Reply:
When publishing to hex it would become `~> 0.4` but to rebar3 it is just "pick the latest". Of course since rebar3 uses lock flies even for deps it does not mean a user of this app would get the latest at the time of pulling it but instead the locked version, or `0.4.0` in this case.

To make this clearer I tend to use like `{telemetry, "~> 0.4"}` in the config.

Reply:
Yeah, I can do one of those. I didn't want to make us push a release for telemetry updates unless the API changed again.